docs/merged_objects.md

```diff
@@ -134,7 +132,9 @@ Each such list will contain the following fields:
 
 | Item name                      | Contents                                                                                                                                                                                                                                                                                                                                                                           |
 | ------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `salmon_version`               | Version of `salmon` used for initial mapping                                                                                                                                                                                                                                                                                                                                       |
+| `sample_id`                    | Sample ID in the form `SCPCS000000`                                                                                                                                                                                                                                                                                                                                                |
+| `library_id`                   | Library ID in the form `SCPCL000000`                                                                                                                                                                                                                                                                                                                                               || `salmon_version`               | Version of `salmon` used for initial mapping
+| `project_id`                   | Project ID in the form `SCPCP000000`                                                                                                                                                                                                                                                                                                                                               || `salmon_version`               | Version of `salmon` used for initial mapping
```

Comment:
```suggestion
| `project_id`                   | Project ID in the form `SCPCP000000`                                                                                                                                                                                                                                                                                                                                               |
```
We shouldn't have this twice?

Reply:
Is this `project_id` or `scpca_project_id`?

Reply:
It's definitely `project_id`: 

```
> metadata(sce)$library_metadata$SCPCL000001 |> names()
 [1] "library_id"              "sample_id"              
 [3] "project_id"              "salmon_version"         
 [5] "reference_index"         "total_reads"            
 [7] "mapped_reads"            "mapping_tool"           
 [9] "alevinfry_version"       "af_permit_type"         
[11] "af_resolution"           "usa_mode"               
[13] "af_num_cells"            "tech_version"           
[15] "assay_ontology_term_id"  "seq_unit"               
[17] "transcript_type"         "include_unspliced"      
[19] "filtering_method"        "miQC_model"             
[21] "prob_compromised_cutoff" "scpca_filter_method"    
[23] "min_gene_cutoff"         "normalization"          
[25] "highly_variable_genes" 
```

But I poked around a little at some files in `scpca-prod` and I see:

1) `SCPCL000001_processed.rds`: `project_id` is in `metadata(sce)`
2) `SCPCL000001_processed_rna.hdf5`: `scpca_project_id` is in `obj.obs`

Those files were both produced with `v0.7.1`. I'm not totally sure where this difference came from...?

Reply:
It's because we add `project_id` to the metadata when we actually create the object using `scpcaTools::read_alevin()` and then when we add the sample metadata to the `colData` prior to AnnData conversion, the columns are coming from `scpca-sample-metadata.tsv`. I just wanted to make sure that discrepancy was correct.

Reply:
> then when we add the sample metadata to the colData prior to AnnData conversion, the columns are coming from scpca-sample-metadata.tsv

Great, this is what I wasn't seeing! The discrepancy is correct.